railties/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Wrap active_record_encryption.configuration on an after initialize
```

Comment:
In the first line here, instead of describing the internals of the change itself, I'd try to clarify what's being fixed here.

Example: Support configuring the digest class in Active Record encryption via initializer.